integration-tests/pom.xml

```diff
@@ -238,21 +238,50 @@
             <id>wls-image-cert</id>
             <properties>
                 <skipITs>false</skipITs>
-                <includes-failsafe>**/ItParameterizedDomain.java,**/ItServerStartPolicy.java,**/ItMiiUpdateDomainConfig.java,**/ItIntrospectVersion.java,**/ItMiiSample.java,**/ItStickySession.java,**/ItSessionMigration.java</includes-failsafe>
+                <includes-failsafe>
+                   **/ItParameterizedDomain.java,
+                   **/ItTwoDomainsLoadBalancers.java,
+                   **/ItServerStartPolicy.java,
+                   **/ItIntrospectVersion.java,
+                   **/ItMiiDomain.java,
+                   **/ItMiiDomainModelInPV.java,
+                   **/ItMiiMultiModel.java,
+                   **/ItMiiUpdateDomainConfig.java,
+                   **/ItMiiSample.java,
+                   **/ItIstioMiiDomain.java,
+                   **/ItIstioDomainInImage.java,
+                   **/ItIstioDomainInPV.java,
+                   **/ItIstioTwoDomainsInImage.java,
+                   **/ItStickySession.java,
+                   **/ItSessionMigration.java,
+                   **/ItMonitoringExporter.java,
+                   **/ItElasticLogging.java,
+                   **/ItCrossDomainTransaction.java,
+                   **/ItPodsRestart.java,
+                   **/ItOperatorRestart.java,
+                   **/ItProductionSecureMode.java,
+                   **/ItExternalRmiTunneling.java,
+                   **/ItKubernetesEvents.java
+                </includes-failsafe>
             </properties>
         </profile>
         <profile>
             <id>fmw-image-cert</id>
             <properties>
                 <skipITs>false</skipITs>
-                <includes-failsafe>**/ItMiiSample.java,**/ItJrfDomainInPV.java</includes-failsafe>
+                <includes-failsafe>
+                  **/ItMiiSample.java,
+                  **/ItFmwDomainInPV.java,
+                  **/ItFmwMiiDomain.java,
+                  **/ItFmwSample.java
+                </includes-failsafe>
             </properties>
         </profile>
         <profile>
             <id>oke-cert</id>
             <properties>
                 <skipITs>false</skipITs>
-                <includes-failsafe>**/ItParameterizedDomain.java,**/ItServerStartPolicy.java,**/ItMiiUpdateDomainConfig.java,**/ItIntrospectVersion.java,**/ItJrfDomainInPV.java,
+                <includes-failsafe>**/ItParameterizedDomain.java,**/ItServerStartPolicy.java,**/ItMiiUpdateDomainConfig.java,**/ItIntrospectVersion.java,**/ItFmwDomainInPV.java,
```

Comment:
can u put this also one class on each line like u did above

Reply:
We do not need java extension.  I updated all the profile with multi line class inclusion without java extension.